### PR TITLE
Updated cloud mailer 

### PR DESCRIPTION
**Pull request by oscarkraemer**
- Removed hardcoded variables
- Added example cfg file
- Made it possible to send a copy to an admin email address

- CCCP-2740

### CODE REVIEW

File: cloudmailer/cloudmailer.py [hunk 1]
```diff
@@ -213,6 +215,36 @@ def getVMsByID(self,uuids):
...
+            if config.has_option('DEFAULT', 'AFFECTED_VMS'):
+                AFFECTED_VMS = config.get('DEFAULT', 'AFFECTED_VMS')
+            if config.has_option('DEFAULT', 'MAIL_SERVER'):
+                MAIL_SERVER = config.get('DEFAULT', 'MAIL_SERVER')
+            if config.has_option('DEFAULT', 'MAIL_FROM'):
+                MAIL_FROM = config.get('DEFAULT', 'MAIL_FROM')
+            if config.has_option('DEFAULT', 'MAIL_BCC'):
+                MAIL_BCC = config.get('DEFAULT', 'MAIL_BCC')
+            if config.has_option('DEFAULT', 'MAX_UPGRADE_AT_ONCE'):
+               MAX_UPGRADE_AT_ONCE = config.get('DEFAULT', 'MAX_UPGRADE_AT_ONCE')
```
Comment (junousi) [inline cloudmailer/cloudmailer.py]: -1 indent

**Comment by junousi** [comment]
This is good for a merge, just want to check the small detail about gitignore.

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,5 @@
+*.swp
+cloudmailer/cloudmailer.cfg
+*.bak
+cloudmailer/temporary_files
+node-drainer/
```
[the remark this answers is not in the document]
Comment (oscarkraemer) [reply .gitignore]: That was a mistake.

File: cloudmailer/cloudmailer.py (hunk 1 above)
Comment (oscarkraemer) [reply cloudmailer/cloudmailer.py]: updated

**Pull request merged**